console/console-init/ui/src/modules/iot-device-detail/components/DeviceDetailHeader/DeviceDetailHeader.tsx

```diff
@@ -221,7 +221,7 @@ const DeviceDetailHeader: React.FunctionComponent<IDeviceDetailHeaderProps> = ({
         id="device-detail-header-kebab-dropdown"
         isPlain={true}
         position={DropdownPosition.right}
-        toggleId="device-detail-header-kebab-toggle"
+        toggleId="device-detail-header-dropdown-toggle"
```

Comment:
to be consistent at all places revisit as id=`device-detail-header-dropdowntoggle`